types/types.d.ts

```diff
@@ -1,7 +1,9 @@
 import { InteractionType } from "discord-interactions"
 
 type DiscordInteractionData = {
-  name: string,
+  command: string,
+  opponent: DiscordInteractionMember,
+  wager: BigInteger,

```

Comment:
As said above, this is apart of Discord's API. We cannot modify these.  Unfortunately, discord types library does not provide a type for this, so we had to create our own simple type here for it.